keepassxc_proxy_client/protocol.py

```diff
@@ -8,19 +8,68 @@
 import nacl.utils
 from nacl.public import PrivateKey, Box, PublicKey
 
+if platform.system() == "Windows":
```

Comment:
Put these things into requirements.txt instead with a requirement conditional https://stackoverflow.com/questions/29222269/is-there-a-way-to-have-a-conditional-requirements-txt-file-for-my-python-applica

Reply:
Done